.github/workflows/pull_request.yml

```diff
@@ -134,15 +134,13 @@ jobs:
         kubectl apply -f https://raw.githubusercontent.com/FoundationDB/fdb-kubernetes-operator/master/config/crd/bases/apps.foundationdb.org_foundationdbrestores.yaml
         # Generate secrets used by the operator
         ./config/test-certs/generate_secrets.bash
-        IMG=localhost:5000/fdb-kubernetes-operator:latest make clean all docker-build docker-push deploy
+        SKIP_TEST=1 IMG=localhost:5000/fdb-kubernetes-operator:latest make clean all docker-build docker-push deploy
```

Comment:
We skip the tests here because we want to deploy the operator first and then pull all tests including the e2e test(s)